src/go/rpk/pkg/cli/cmd/container/start.go

```diff
@@ -279,18 +287,21 @@ func restartCluster(c common.Client) ([]node, error) {
 			mu.Lock()
 			nodes = append(nodes, node{
 				state.ID,
-				fmt.Sprintf(
-					"%s:%d",
-					state.ContainerIP,
-					state.HostKafkaPort,
-				),
+				nodeAddr(state.HostKafkaPort),
 			})
 			mu.Unlock()
 			return nil
 		})
 	}
 	err = grp.Wait()
-	return nodes, err
+	if err != nil {
+		return nil, fmt.Errorf("Error restarting the cluster: %v", err)
+	}
+	err = waitForCluster(check(nodes))
+	if err != nil {
+		return nil, err
```

Comment:
I think this would be enough: https://github.com/vectorizedio/redpanda/pull/430/files#diff-24a1b5b910769f8aaa14ada82b9d18cec7b5d1618db88b23dc2c98275d582283R331

Reply:
I was triggered by [this error ](https://github.com/vectorizedio/redpanda/pull/430/files#diff-24a1b5b910769f8aaa14ada82b9d18cec7b5d1618db88b23dc2c98275d582283R326).
In that case we could end up with a more cryptic error. But it should be fine both ways to be honest.